addons/kafka.md

```diff
@@ -137,6 +137,12 @@ These variables are made available inside the pod.
 | `KAFKA_CERTIFICATE` | Client certificate for connecting to the Kafka brokers, as string data |
 | `KAFKA_PRIVATE_KEY` | Client certificate key for connecting to the Kafka brokers, as string data |
 | `KAFKA_CA` | Certificate authority used to validate the Kafka brokers, as string data |
+| `KAFKA_KEYSTORE` | PKCS#12 keystore for use with Java clients |
```

Comment:
Do we really want to expose these as strings? I don't think it adds any value.

Reply:
I tend to agree. We will need to change their keys to something that is invalid as env-variables for them not to be included by default, so that would need to be fixed in nais/kafkarator#3. I'll do that along with other fixes there.

Reply:
Fixed